packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -0,0 +1,1297 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import static groovy.io.FileType.FILES
+
+import com.android.build.OutputFile
+import groovy.json.JsonSlurper
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+import org.apache.tools.ant.taskdefs.condition.Os
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.Project
+import org.gradle.api.Plugin
+import org.gradle.api.Task
+import org.gradle.api.file.CopySpec
+import org.gradle.api.file.FileCollection
+import org.gradle.api.logging.LogLevel
+import org.gradle.api.tasks.Copy
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.OutputFiles
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.bundling.Jar
+import org.gradle.internal.os.OperatingSystem
+import org.gradle.util.VersionNumber
+
+/**
+ * For apps only. Provides the flutter extension used in app/build.gradle.
+ * The versions specified here should match the values in
```

Comment:
nit:
```suggestion
 * For apps only. Provides the flutter extension used in app/build.gradle.
 *
 * The versions specified here should match the values in
```

This isn't getting machine-consumed [like dartdoc](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements), but I think the short self-contained summary paragraph at the top is a helpful pattern for human readers seeing this in the source, too.

Reply:
Right, I agree!